src/cmd/run.js

```diff
@@ -69,6 +73,10 @@ export default async function run(
 
   const manifestData = await getValidatedManifest(sourceDir);
 
+  if (lint) {
+    await linter({sourceDir, artifactsDir, ignoreFiles});
```

Comment:
could we pass the lint and linter parameters to the `MultiExtensionRunner` instance and leave it to handle both the initial linting and the linting on file changes?